test/train-sets/ref/0001-replay.stderr

```diff
@@ -18,18 +18,18 @@ loss     last          counter         weight          label        predict feat
 0.250000 0.250000            8            8.0         0.0000         0.0000      860
 0.312500 0.375000           16           16.0         1.0000         0.0000      128
 0.343750 0.375000           32           32.0         0.0000         0.0000      176
-0.350946 0.358141           64           64.0         0.0000         0.0275      350
-0.371181 0.391417          128          128.0         1.0000         0.1510      620
-0.292717 0.214252          256          256.0         0.0000         0.3660      410
-0.149202 0.005688          512          512.0         0.0000         0.0000      278
-0.074603 0.000004         1024         1024.0         1.0000         1.0000      170
+0.359375 0.375000           64           64.0         0.0000         0.0275      350
+0.414062 0.468750          128          128.0         1.0000         0.1510      620
+0.433594 0.453125          256          256.0         0.0000         0.3660      410
+0.441406 0.449219          512          512.0         0.0000         0.0000      278
+0.453125 0.464844         1024         1024.0         1.0000         1.0000      170
 
 finished run
 number of examples per pass = 200
 passes used = 8
 weighted example sum = 1600.000000
 weighted label sum = 728.000000
-average loss = 0.047746
+average loss = 0.455000
```

Comment:
so loss went up?

Reply:
Previous loss was cheating since it was using predictions that were coming out of learn.